### ISSUE TITLE
Table stopped working

### ISSUE THREAD

**Issue by marcelinorodrigom** [google/google-visualization-issues#2824]
We're having some issues to render a table using the API:

```
VM831:1 Uncaught SyntaxError: Unexpected token v in JSON at position 1
    at JSON.parse (<anonymous>)
    at gvjs_Ii (jsapi_compiled_default_module.js:64)
    at gvjs_cm.gvjs_.Fwa (jsapi_compiled_default_module.js:244)
    at gvjs_cm.<anonymous> (jsapi_compiled_format_module.js:45)
    at gvjs_Wk (jsapi_compiled_default_module.js:155)
    at gvjs_3l.gvjs_.dispatchEvent (jsapi_compiled_default_module.js:153)
    at gvjs_8l (jsapi_compiled_default_module.js:235)
    at gvjs_3l.gvjs_.uua (jsapi_compiled_default_module.js:234)
    at gvjs_3l.gvjs_.Sca (jsapi_compiled_default_module.js:234)
```

It was working until early May (using the version 1), then it started working intermittently (which seems to be related to whether the redirection got HTTP 200), but now, it won't work at all.

We tried changing to a more recent version, tried to change the code to use the new loader, but with no success.

P.S.: I noticed that the issue #2821 had a similar error message, although related to GeoChart.

We'd appreciate any help!

**Comment by dlaliberte** [comment]
The redirection of jsapi to the new loader is back to 100%, and we would like to leave it in place if possible.   This redirect means you will be using a newer version as well, unless you change the version to something like '45.2'.  

I can't tell why you are getting a JSON parse error without details but I would guess it probably involves something in how you are fetching your data.

**Comment by spleenware** [comment]
I'm not sure if this is related, but I'm now getting 404 errors in my app (using the data tables) on this resource:
https://www.gstatic.com/charts/48.1/js/jsapi_compiled_corechart%2Ctable_module.js

**Comment by dlaliberte** [comment]
That is interesting, @spleenware, since the "corechart%2Ctable" indicates that comma (',' is %2C) got encoded as part of a single conjoined package name rather than 2 package names separated by comma.  You should be loading packages: ['corechart', 'table'].  If you are, then there is a bug in the loader.  Are you using the autoload feature?

**Comment by spleenware** [comment]
> That is interesting, @spleenware, since the "corechart%2Ctable" indicates that comma (',' is %2C) got encoded as part of a single conjoined package name rather than 2 package names separated by comma. You should be loading packages: ['corechart', 'table']. If you are, then there is a bug in the loader. Are you using the autoload feature?

It's been SO long since I wrote the app, but here's the page snippet:


  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript" src="/tablequerywrapper.js"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {'packages' : ['corechart,table']});
    google.setOnLoadCallback(init);

**Comment by dlaliberte** [comment]
Right.. So the jsapi loader had this additional interesting feature of splitting the list of packages.  I may add that to the new loader, but in the meantime, split those packages up manually.  Should be {'packages' : ['corechart', 'table']}.

Also, please switch to using the new loader:

https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code

**Comment by spleenware** [comment]
> Right.. So the jsapi loader had this additional interesting feature of splitting the list of packages. I may add that to the new loader, but in the meantime, split those packages up manually. Should be {'packages' : ['corechart', 'table']}.
> 
> Also, please switch to using the new loader:
> 
> https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code

I can't actually rebuild this project any more, so it's stuck the way it is :-(. (ie. some of the toolchains are now gone)

**Comment by dlaliberte** [comment]
Sorry.. @marcelinorodrigom.  I forgot the OP was about a different issue.  Curious to find out your details.

**Comment by spleenware** [comment]
> Right.. So the jsapi loader had this additional interesting feature of splitting the list of packages. I may add that to the new loader, but in the meantime, split those packages up manually. Should be {'packages' : ['corechart', 'table']}.
> 
> Also, please switch to using the new loader:
> 
> https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code

Should I raise a separate issue and beg (please, please) for the legacy load() to be supported again! I'm really in a pickle.

**Comment by dlaliberte** [comment]
Sigh.. I can add this feature to the new loader.  The old loader on its last legs.

**Comment by spleenware** [comment]
> Sigh.. I can add this feature to the new loader. The old loader on its last legs.

Very much appreciated. Thank you.
